go.mod

```diff
@@ -1,25 +1,18 @@
 module github.com/liftbridge-io/liftbridge
 
-go 1.12
+go 1.13
 
 require (
 	github.com/Workiva/go-datastructures v1.0.50
 	github.com/dustin/go-humanize v1.0.0
-	github.com/golang/protobuf v1.3.1
+	github.com/golang/protobuf v1.3.2
 	github.com/hako/durafmt v0.0.0-20190612201238-650ed9f29a84
-	github.com/hashicorp/go-hclog v0.9.2 // indirect
```

Comment:
All of these were removed when I ran `go mod tidy`. Don't seem to be necessary to build the project? 🙈